tools/integration_tests/benchmarking/setup_test.go

```diff
@@ -58,8 +68,8 @@ func TestMain(m *testing.M) {
 	setup.SetUpTestDirForTestBucketFlag()
 
 	flagsSet := [][]string{
-		{"--stat-cache-ttl=0"},
-		{"--client-protocol=grpc", "--stat-cache-ttl=0"},
+		{"--stat-cache-ttl=0", "--enable-atomic-rename-object=true"},
```

Comment:
Instead, can we do similar to `managed_folder` e2e test, where all the file (test_suites) will have their own mount configuration and expected latency?

Reply:
It required changes in all the files that's why I avoid initially. Anyways fixed it for all the existing tests also.